cmd/clusterregistry/BUILD.bazel

```diff
@@ -11,6 +11,28 @@ docker_push(
     name = "push-clusterregistry-image",
     image = ":clusterregistry-image",
     registry = "gcr.io",
-    repository = "kubetest-157500/clusterregistry",
+    repository = "$(project)/clusterregistry",
     tag = "dev",
 )
+
```

Comment:
This bit should not be checked in, since it's autogenerated: can you revert this?